testing/novnc/APKBUILD

```diff
@@ -0,0 +1,20 @@
+# Contributor: Andre Bierwolf <a.b.bierwolf@gmail.com>
+# Maintainer: Andre Bierwolf <a.b.bierwolf@gmail.com>
+pkgname=novnc
+_pkgname=noVNC
+pkgver=0.6.2
+pkgrel=0
+pkgdesc="VNC client using HTML5 (WebSockets, Canvas) with encryption (wss://) support."
+url="https://kanaka.github.io/noVNC/"
+arch="noarch"
+license="MPL-2.0"
+source="$pkgname-$pkgver.tar.gz::https://github.com/$pkgname/$_pkgname/archive/v$pkgver.tar.gz"
+options="!check"
+
+package() {
+        mkdir -p "$pkgdir/usr/share/novnc"
+        cp -r $srcdir/$_pkgname-$pkgver/* $pkgdir/usr/share/novnc
```

Comment:
`$srcdir` and `$pkgdir` should be quoted (like `"$srcdir"/$_pkgname-$pkgver/*`).

Please use tabs for indentation.